### PR TITLE
Newsletter-281-Translate-into-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-12-13-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+*Dans cette rubrique mensuelle, nous mettons en évidence les mises à jour
+intéressantes des portefeuilles et services Bitcoin.*
+
+- **Lancement d'un pool de minage Stratum v2 :**
+  [DEMAND][demand website] est un pool de minage construit à partir de l'[implémentation de référence Stratum v2][news247 sri]
+  permettant initialement le minage en solo, avec le minage en pool prévu pour l'avenir.
+
+- **Annonce de l'outil de simulation du réseau Bitcoin warnet :**
+  Le [logiciel warnet][warnet github] permet de spécifier des topologies de nœuds, d'exécuter des scénarios scriptés sur ce réseau et
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-12-13-newsletter.md:46]: missing links to "scripted scenarios" and "monitoring"

File: _posts/fr/newsletters/2023-12-13-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  de surveiller et analyser les comportements qui en résultent.
+
+- **Publication d'un client Payjoin pour Bitcoin Core :**
+  Le [payjoin-cli][] est un projet en Rust qui ajoute des fonctionnalités d'envoi et de réception de [payjoin][topic payjoin] en ligne
+  de commande pour Bitcoin Core.
+
+- **Appel aux horodatages d'arrivée des blocs par la communauté :**
+  Un contributeur au référentiel [Bitcoin Block Arrival Time Dataset][block arrival github] a appelé les opérateurs de nœuds à soumettre
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-12-13-newsletter.md:54]: Missing link to tweet

File: _posts/fr/newsletters/2023-12-13-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  leurs horodatages d'arrivée de bloc à des fins de recherche. Il existe un référentiel similaire pour collecter des [données sur les
+  blocs obsolètes][stale block github].
+
+- **Envoy 1.4 publié :**
+  La version 1.4 du portefeuille Bitcoin Envoy ajoute notamment la [gestion des pièces][topic coin selection] et l'[étiquetage des
+  portefeuilles][topic wallet labels] (BIP329 à venir), ainsi que d'autres fonctionnalités.
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-12-13-newsletter.md:60]:
```suggestion
  portefeuilles][topic wallet labels] ([BIP329][] à venir), ainsi que d'autres fonctionnalités.
```

File: _posts/fr/newsletters/2023-12-13-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+  Le [payjoin-cli][] est un projet en Rust qui ajoute des fonctionnalités d'envoi et de réception de [payjoin][topic payjoin] en ligne
+  de commande pour Bitcoin Core.
+
+- **Appel aux horodatages d'arrivée des blocs par la communauté :**
+  Un contributeur au référentiel [Bitcoin Block Arrival Time Dataset][block arrival github] a appelé les opérateurs de nœuds à soumettre
+  leurs horodatages d'arrivée de bloc à des fins de recherche. Il existe un référentiel similaire pour collecter des [données sur les
+  blocs obsolètes][stale block github].
+
+- **Envoy 1.4 publié :**
+  La version 1.4 du portefeuille Bitcoin Envoy ajoute notamment la [gestion des pièces][topic coin selection] et l'[étiquetage des
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-12-13-newsletter.md:59]: Missing link to release notes

File: _posts/fr/newsletters/2023-12-13-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  portefeuilles][topic wallet labels] (BIP329 à venir), ainsi que d'autres fonctionnalités.
+
+- **Annonce du schéma d'encodage BBQr :**
+  Le [schéma][bbqr github] peut encoder efficacement des fichiers plus volumineux, tels que des [PSBT][topic psbt], en une série de
+  QR animés pour une utilisation dans des configurations de portefeuille hors ligne.
+
+- **Zeus v0.8.0 publié :**
+  La version v0.8.0 de Zeus contient un nœud LND intégré, un support supplémentaire pour les [canaux zero conf][topic zero-conf channels]
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-12-13-newsletter.md:67]: Missing link to release notes

**Review by murchandamus — COMMENTED** [review]
LGTM, ACK with nit:

I tested all links and compared the translation visually with the English original. I noticed that a few links were present in the English newsletter, but not in the French. In case they weren’t left out on purpose, here they are:

**Comment by bitschmidty** [comment]
> So sad I made a mistake and some other to solve the first, and now I don't know how to solved it ! :,(
> 
> please @murchandamus can you help me ?

See my comment here. Fix for CI is WIP.

https://github.com/bitcoinops/bitcoinops.github.io/pull/1435#issuecomment-1861334521

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin 

Ran checks locally since travis isnt working atm

**Comment by Copinmalin** [comment]
it wasn't my mistake then? so much the better, 
thanks for your help

**Pull request merged**